File: Pattern28.py
# E
# D E
# C D E
# B C D E
# A B C D E

n = int(input("Enter the number:"))
for i in range(0,n):
    val = n - i
    for j in range(0,i+1):
        print(chr(64+val+j)," " ,end='')
    print()

File: PATTERNS/Pattern28/Pattern28.java
import java.util.Scanner;

// E
// D E
// C D E
// B C D E
// A B C D E

public class Pattern28 {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n;
        System.out.print("Enter the number:");
        n = in.nextInt();
        for (int i = 0; i < n; i++) {
            int val = n - i;
            for (int j = 0; j <= i; j++) {
                System.out.printf("%c ", (char) (64 + val + j));
            }
            System.out.printf("\n");
        }
    }
}